src/domain/cart/CartPage.tsx

```diff
@@ -1,5 +1,74 @@
-export const CartPage = (): JSX.Element => {
+import { cartSelector, removeFromCart, getTotals } from "./cartSlice";
+import { Link } from "react-router-dom";
+import { CartItem } from "./types";
+import { useEffect } from "react";
+import { ToastContainer} from 'react-toastify'
+import { useAppDispatch, useAppSelector } from "../../store.hooks";
+const CartPage = (): JSX.Element => {
+  const cart = useAppSelector(cartSelector);
+  const dispatch = useAppDispatch();
+
+  const handleRemoveFromCart = (cartItem: CartItem) => {
+    dispatch(removeFromCart(cartItem));
+  };
+
+  useEffect(() => {
+    dispatch(getTotals());
```

Comment:
getTotals() should not be in redux. Instead, put in a utils file and import it here.
Also, the useEffect here is redundant. You can call the fn where you need its value and if it's an expensive fn then use useMemo.